docs/docs/references/users/README.md

```diff
@@ -1,5 +1,95 @@
 # 👨‍👩‍👧‍👦 Users
 
+The _users_ are the main entities of the identity service.
+We will describe the user-related concepts and details in the following.
+
 ## Profile
 
+Each user has a profile containing the user information, more than the name and contact details.
+
+A user profile consists of 3 sections:
```

Comment:
not sure should it be 3 sections or three sections? Also, this might be confusing, from the data schema hard to tell a three-section structure right?

Reply:
maybe follow Auth0?

<img width="601" alt="image" src="https://user-images.githubusercontent.com/10594507/176587119-be536511-7eb6-429b-9a80-07c7b20b8232.png">

Reply:
> Also, this might be confusing, from the data schema hard to tell a three-section structure right?

Yes. The following list will tell, is it enough for readers to understand?